src/Microsoft.AspNet.Mvc.Core/Rendering/Html/TagBuilder.cs

```diff
@@ -117,7 +119,7 @@ private void AppendAttributes(StringBuilder sb)
                     continue;
                 }
 
-                var value = WebUtility.HtmlEncode(attribute.Value);
+                var value = _htmlEncoder.HtmlEncode(attribute.Value);
```

Comment:
:ok: please comment when you've added (an) additional commit(s) to cover the still-open comments